pkg/virt-handler/vm.go

```diff
@@ -598,6 +598,13 @@ func (c *VirtualMachineController) Run(threadiness int, stopCh chan struct{}) {
 	defer c.Queue.ShutDown()
 	log.Log.Info("Starting virt-handler controller.")
 
+	// Make sure the tun module is loaded on the node
```

Comment:
@jean-edouard I think a better place for this in-direct probing of TUN kernel module will be in the file cmd/virt-handler.go because looks like most of the initialization is done there. Another thing is that it may require a functional test where you unload the TUN module on a specific node and then re-create virt-handler pod and see if the TUN kernel module appears again.

Reply:
Regarding a functional test. We've discussed this in the past. On a system without selinux, libvirt (that's running in the virt-launcher pod) will do the work. It probes the module by default.
However, on a system with enabled selinux, virt-launcher will run as `conatiner_t` and will not be allowed to probe the kernel module, that's why this change was posted.

I think the best test is simply a running VMI on selinux enabled lane.

Reply:
I'm uncomfortable with the notion of unloading kernel modules in automated tests. Can we guarantee this won't disrupt the system itself? other tests caught in the crossfire?

Reply:
@enp0s3 good points, I'll look at cmd/virt-handler.go. I didn't know what kind of test to write but that's a good idea.
@vladikr +1
@stu-gott is this a question about the test suggested by @enp0s3? This PR only triggers loads, no unloads.

Reply:
@jean-edouard yes that was my concern. I don't want to increase the scope of this PR to cover that.